doc/changes/latest.rst

```diff
@@ -43,6 +43,9 @@ Enhancements
 
 Changes
 -------
+- :bdg-dark:`Code` Warn the user :func:`nilearn.interfaces.fmriprep.load_confounds_scrub.extract_outlier_regressors` if ``sample_mask`` is empty (all volumes are scrubbed) (:gh:`4558` by `Victoria Shevchenko`_).
```

Comment:
```suggestion
- :bdg-dark:`Code` Warn the user when all volumes would be scrubbed when loading fmriprep confounds as this would lead to an empty ``sample_mask`` (:gh:`4558` by `Victoria Shevchenko`_).
```

Reply:
I double checked and extract_outlier_regressors is not in the public API of nilearn and that's why sphinx cannot find a reference to it and is throwing an error in the doc build. Not mentionning the exact function should fix it.